Domains/Core/BisCore.ecschema.xml

```diff
@@ -1868,6 +1868,7 @@
 
     <ECEntityClass typeName="PhysicalMaterial" modifier="Abstract" displayLabel="Physical Material" description="A bis:PhysicalMaterial defines the matter of which physical objects are made. bis:PhysicalMaterials are not used to define rendering characteristics (see bis:RenderMaterial).">
         <BaseClass>DefinitionElement</BaseClass>
+        <ECProperty propertyName="Density" typeName="double" description="Density" displayLabel="Density" minimumValue="0.0" kindOfQuantity="AECU:DENSITY" />

```

Comment:
do not duplicate property name in displayLabel if the invariant name is the same as the display label.  Also do not set the description to the display label or invariant name.  Only set the description if it adds meaning.

Consider writing some docs in the remarks.md to add detailed description of Density

Reply:
@ColinKerr Thank you - taking care of both comments r.n.